### PR TITLE
Sephirah (Admin-Enabled, Highpop Role)

### PR DESCRIPTION
**Pull request by Kitsunemitsu**
## About The Pull Request
Testing a Staff Officer Like role, the Sephirah, for highpop.
They cannot use EGO at all, and have their own little room on the manager's floor for Overwatch.

## Why It's Good For The Game
This has been tested on 2023/02/04, and feedback was extremely positive.
First vote gave 12-1 positive/negative feedback and the second vote was 7-0.
There was 20-35 players, and Roleplay was heavily increased during these rounds.
As well as that, agent coordination was dramatically increased; with a heavily increased use of both architecture and departmental communications.

I would like to look into activating this role during highpop in the future.

## Changelog
:cl:
add: Added sephirah as an admin-enabled role.
add: Added sephirah.txt, a small dictionary of 8 sephirah names lifted from the bible
add: Added sephirah overwatch area and command canteen
del: Removed Clown names from the preferences menu
/:cl:

### CODE REVIEW

**Comment by EgorDinamit** [comment]
I see no reason for this to exist even on 30+ pop, but you do you.

**Comment by Kitsunemitsu** [comment]
I will not merge this PR.

File: code/game/machinery/computer/crew.dm [hunk 1]
```diff
@@ -35,7 +35,8 @@ GLOBAL_DATUM_INIT(crewmonitor, /datum/crewmonitor, new)
 		// 00: Manager
 		"Manager" = 00,
 		"Extraction Officer" = 01,
-		"Records Officer" = 01,
+		"Records Officer" = 02,
```
Comment (SpaceJohn413) [inline code/game/machinery/computer/crew.dm:38]: 2 ROs?

Comment (Kitsunemitsu) [reply code/game/machinery/computer/crew.dm:38]: It's just where they are on the crew monitor console

**Review by dsmith328 — COMMENTED** [review]
Looks good but I'd like to personally see it in action, ping me on the discord when you next test this I'm _very_ interested.

**Comment by Kitsunemitsu** [comment]
> Looks good but I'd like to personally see it in action, ping me on the discord when you next test this I'm _very_ interested.

Look to be doing one more test tomorrow; will notify you.

File: code/modules/jobs/job_types/agent.dm [hunk 2]
```diff
@@ -61,6 +61,10 @@
 			ears = /obj/item/radio/headset/headset_records
 			accessory = /obj/item/clothing/accessory/armband/lobotomy/records
 
+		else	//Pick a department or get training.
```
Comment (EgorDinamit) [inline code/modules/jobs/job_types/agent.dm:64]: If that's the case, remove the Training from the switch itself.

Comment (Kitsunemitsu) [reply code/modules/jobs/job_types/agent.dm:64]: Good point; i overlooked that.

**Review by dsmith328 — DISMISSED** [review]
From what I saw, this massively improved engagement in radio chatter and also encouraged more organization. People seemed to really like it overall and it seems to have ONLY been positive as an impact. It not being always enabled also minimizes any issue it may have from being an excessive role in the list.

**Review by dsmith328 — APPROVED** [review]
Ye

**Pull request merged**